src/main/java/org/sonarsource/scanner/cli/Cli.java

```diff
@@ -24,6 +24,7 @@
 
 import static java.util.Arrays.asList;
 
+// TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.scanner.cli%3Asonar-scanner-cli&branch=feature%2Fjanos%2Fjunk1&issues=AWFQYVNasS4rYBZqE3DX&open=AWFQYVNasS4rYBZqE3DX)

[ISSUE_KEY]: AWFQYVNasS4rYBZqE3DX